tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs

```diff
@@ -84,6 +84,12 @@ public void ByteOrder(string imagePath, ByteOrder expectedByteOrder)
         public void TiffDecoder_CanDecode_Uncompressed<TPixel>(TestImageProvider<TPixel> provider)
             where TPixel : unmanaged, IPixel<TPixel> => TestTiffDecoder(provider);
 
+        [Theory]
+        [WithFile(FlowerRgb888Planar6Strips, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb888Planar15Strips, PixelTypes.Rgba32)]
+        public void TiffDecoder_Planar<TPixel>(TestImageProvider<TPixel> provider)
```

Comment:
new test files are good,
we needs to add more test files for tiff

Reply:
we had test files for planar, but only with 3 strips, so the bug was not triggered.